server/config/toml.go

```diff
@@ -237,9 +237,27 @@ snapshot-interval = {{ .StateSync.SnapshotInterval }}
 snapshot-keep-recent = {{ .StateSync.SnapshotKeepRecent }}
 
 ###############################################################################
-###                         Store / State Streaming                         ###
+###                              State Streaming                            ###
 ###############################################################################
 
+# Streaming allows nodes to stream state to external systems
+[streaming]
+
+# streaming.abci specifies the configuration for the ABCI Listener streaming service
+[streaming.abci]
+
+# List of kv store keys to stream out via gRPC
+# Set to ["*"] to expose all keys.
+keys = [{{ range .Streaming.ABCI.Keys }}{{ printf "%q, " . }}{{end}}]
```

Comment:
I'm a little confused on what the key values would be. Are they basically the module names? Or are they something else? It'd be nice to have an example or two (other than `"*"`).

Also, all the comments in this newly added section need periods.

Reply:
They're module `StoreKey` names. I'll update the comment to include examples and punctuation.

Reply:
fixed punctuation and updated comment.